src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDynamicMemberReferenceExpression.cs

```diff
@@ -0,0 +1,374 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [Fact]
+        public void IDynamicMemberReferenceExpression_SimplePropertyAccess()
+        {
+            string source = @"
+using System;
+
+namespace ConsoleApp1
+{
+    class C1
+    {
+        static void M1()
+        {
+            dynamic d = null;
+            int i = /*<bind>*/d.Prop1/*</bind>*/;
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IDynamicMemberReferenceExpression (Member name: Prop1, Containing Type: null) (OperationKind.DynamicAccessExpression, Type: dynamic) (Syntax: 'd.Prop1')
+  Instance Receiver: ILocalReferenceExpression: d (OperationKind.LocalReferenceExpression, Type: dynamic) (Syntax: 'd')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<MemberAccessExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact]
+        public void IDynamicMemberReferenceExpression_InvalidPropertyAccess()
+        {
+            string source = @"
+using System;
+
+namespace ConsoleApp1
+{
+    class C1
+    {
+        static void M1()
+        {
+            dynamic d = null;
+            int i = /*<bind>*/d./*</bind>*/;
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IDynamicMemberReferenceExpression (Member name: , Containing Type: null) (OperationKind.DynamicAccessExpression, Type: dynamic, IsInvalid) (Syntax: 'd./*</bind>*/')
```

Comment:
Member name should be printed as `""`, check for other places in the dumper where we log strings.